sceptre/scipool/config/strides/guardduty-member.yaml

```diff
@@ -1,10 +0,0 @@
-template_path: remote/GuardDutyMember.yaml
-stack_name: guardduty-member
-dependencies:
-  - strides/essentials.yaml
```

Comment:
I believe we'll still need sceptre to deploy DG to strides accounts... True?

Reply:
i guess if you want keep guardduty on strides then yes.  I have restored the file.

Reply:
it turns out that this file sets up guardduty member with the guardduty master that was setup with sceptre.  since we are going to setup a new master this will no longer work.  We need to remove then re-add and associate with a new master.